charts/nr-k8s-otel-collector/templates/deployment-configmap.yaml

```diff
@@ -10,14 +10,14 @@ data:
     receivers:
       otlp:
         protocols:
-          {{ if .Values.enableGrpcGatewayEndpoint }}
+          {{ if .Values.gateway.grpc.enable }}
           grpc:
             endpoint: ${env:MY_POD_IP}:4317
           {{ end }}
```

Comment:
If there will be a future PR to templatise the configuration I would remove as many toggles as possible and you decide the templates and the tree of values.

When I [said this](https://github.com/newrelic/helm-charts/pull/1342#discussion_r1594013650) was to give you an example, not encourging you to change it but to follow [Paolo's suggestion](https://github.com/newrelic/helm-charts/pull/1342#discussion_r1593583200) of removing it.

If it is not used yet it makes no sense to support it.

Reply:
We talked about it and we agree that it doesn't make sense to keep it in unused. We've gone ahead and removed it. Thanks!